src/providers/sidebarProvider.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+import React, { createContext, ReactNode, useState } from 'react';
+
+export interface SidebarContextProps {
```

Comment:
i'd prefer to move it to folder `types`.

Reply:
@vickyadrii done kaks